pkgs/top-level/all-packages.nix

```diff
@@ -111,6 +111,16 @@ with pkgs;
 
   nix-update-script = callPackage ../common-updater/nix-update.nix { };
 
+  ### Push NixOS images into the fixed point
+
+  nixosImages = let
+    system = stdenv.hostPlatform.system;
+    release-packages = (import ../../nixos/release.nix) {
+      supportedSystems = [ stdenv.hostPlatform.system ];
+    };
+  in lib.mapAttrs (name: value: value.${system})
+    { inherit (release-packages) iso_minimal iso_gnome iso_plasma5 amazonImage; };
```

Comment:
These underscores are counter to Nixpkgs conventions.

Does it make sense to use `.` here? If we use the same config to generate multiple images, it'd make sense to have `minimal.iso` for example. Otherwise, `isoMinimal` is fine.

Reply:
I think something like `isos.minimal` would be better. But that should be done in another PR. This is trying to tackle, how to test images from packages.